### ISSUE TITLE
Integration Tests for attack.py

### ISSUE THREAD

**Issue by abheesht17** [SforAiDl/decepticonlp#78]
I am clueless about how to implement integration tests. Otherwise, I have run the code on datasets for all the four different classification losses on my local repo.

**Comment by rajaswa** [comment]
Sure. I can try and help.
Meanwhile, we can add example scripts, if not tests?
 ```
examples/attacks/run_attack_imdb.py
```

**Comment by abheesht17** [comment]
Yes, sure, I have a couple of examples which I tried. I'll put them up
